### ISSUE TITLE
Deactivate update_addons_current_version/fast_current_version crons

### ISSUE THREAD

**Issue by diox** [mozilla/olympia#1242]
This is mostly lifted from https://bugzilla.mozilla.org/show_bug.cgi?id=956752 but re-explained with the latest investigation results and outside of bugzilla.

The `update_addons_current_version` / `fast_current_version` crons are nasty, going over all the database and spawning many tasks in an attempt to correct an old issue with the `_current_version` denormalized field on addons.

A long time ago the AMO team figured they should be useless by now and added a waffle switch to turn them off. Unfortunately, once that switch was flipped, regressions happened and it was never changed since - the switch is *on* (at least in prod), which leaves the crons enabled.

From looking at Kibana it looks like these crons are useless nowadays, at least for non-personas. If I query for `"Version changed from current" AND (Hostname:"ip-172-31-47-71" OR Hostname:"ip-172-31-47-72" OR Hostname:"ip-172-31-37-161" OR Hostname:"ip-172-31-37-163")` in prod, I get no results, indicating that this log statement is never hit from celery ips. That statement is inside `Addon.update_version()`, which the cron task calls directly - that's all the task does, really. If you remove the hostname filtering you can see results, the log statement is hit regularly from our webservers when new versions are submitted/reviewed.

**TL;DR:** those crons can probably be deactivated.

Let's test that theory by turning the `current_version_crons` waffle switch *off* in dev/stage and then prod soon.

**Comment by andymckay** [comment]
We should be able to cope with this in our infrastructure though, so this does feel like a big risk. If it fails what would we expect to see, what's our STR so that Krupa can test?

**Comment by diox** [comment]
STR:
- Submit a new addon, see if it appears correctly (including version information everywhere)
- Submit a new version, see if it appears correctly (including version information everywhere)
- Push a newly submitted version to public, see if it appears correctly (including version information everywhere)

Etc. Basically playing with versions :)

The risk is minimal because those crons are here as a fallback to set `current_version` when it's somehow not set correctly. Because of this, just re-enabling them will fix any issues. In fact, we can use that in our advantage to test what happens on stage : after playing with addons with the crons turned off, we can turn them on, wait for them to run, and look again in the logs to see if they did anything.

**Comment by diox** [comment]
Switch was turned off Jan. 12, 2016, 10:23 a.m. UTC on dev.

**Comment by diox** [comment]
Switch was turned off Jan. 19, 2016, 9:50 a.m. UTC on stage.

No issues were found on dev or stage during that period.
